EventFilter/CTPPSRawToDigi/src/RawDataUnpacker.cc

```diff
@@ -275,6 +277,14 @@ int RawDataUnpacker::ProcessVFATDataParallel(const uint16_t *buf, unsigned int O
 
   if (hFlag == vmRaw)
     wordsProcessed += 9;
+  
+  if (hFlag == vmDiamondCompact)
```

Comment:
case or else-if for the vmCluster, vmRaw, vmDiamondCompact seems more appropriate